### ISSUE TITLE
docs: Create documentations for the available commands

### ISSUE THREAD

**Issue by Twaha-Rahman** [freeCodeCamp/freeCodeCamp#50992]
Currently we have a lot of available commands in the project but most, if not all, contributors have little to no idea what they do. In light of the issue #50991, I think it's important we create some documentations for these commands. That way contributors can know what these commands do and we can prevent issues like #50991.

![Screenshot from 2023-07-17 12-21-03](https://github.com/freeCodeCamp/freeCodeCamp/assets/39026437/a232983c-4715-4abd-8ed9-17862f3d7096)
![Screenshot from 2023-07-17 12-21-41](https://github.com/freeCodeCamp/freeCodeCamp/assets/39026437/ff3c381b-5598-4771-b756-a1742012cb4a)

**Comment by Twaha-Rahman** [comment]
Btw... If anyone know this, then please do tell me: How do you serve the production build? 

(Created the prod build with `pnpm run build`)

**Comment by ShaunSHamilton** [comment]
> How do you serve the production build?

```bash
serve -c serve.json
```
> https://contribute.freecodecamp.org/#/devops?id=first-install-2

**Comment by hafeeluddeen** [comment]
hi, can I work on this?

**Comment by naomi-lgbt** [comment]
TBH, I think the bulk of these are commands that contributors will rarely, if ever, run.

I'd rather focus on keeping the documentation up to date in the step-by-step guide format it already is.

**Comment by Twaha-Rahman** [comment]
> TBH, I think the bulk of these are commands that contributors will rarely, if ever, run.

I agree that most people wouldn't need them but still it's best to have them documented IMO. But I do admit keeping the docs up-to-date is going to challenge. 

> I'd rather focus on keeping the documentation up to date in the step-by-step guide format it already is.

But there's one thing that needs some clarification IMO. The [page](https://contribute.freecodecamp.org/#/how-to-add-cypress-tests) in the docs explaining how to run the Cypress tests needs some more explanations because some of the tests fail to run because Mailhog wasn't configured properly.

Maybe a small reminder saying that they should set up Maihog before running the Cypress tests otherwise expect the tests for the emails to fail.

**Comment by ARMummert** [comment]
I would like to work on this issue is it still available?
